eng/scripts/Language-Settings.ps1

```diff
@@ -87,7 +87,11 @@ function Get-javascript-PackageInfoFromPackageFile ($pkg, $workingDirectory)
 
   $packageJSON = ResolvePkgJson -workFolder $workFolder | Get-Content | ConvertFrom-Json
   $pkgId = $packageJSON.name
-  $docsReadMeName = $pkgId -replace "^@azure/" , ""
+  # regex generated readme name as follows
+  # @azure/core-client  -> core-client
+  # @azure-rest/client  -> rest-core-client
```

Comment:
```suggestion
  # @azure-rest/client  -> core-client-rest
```
Don't we want that so that we match the folder name?

Reply:
We should follow the approach Wes outlines above. We already have readme.md files published using the `@azure-rest/example-package` -> `example-package-rest-readme.md` ([example](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/main/docs-ref-services/preview/core-client-paging-rest-readme.md))